functions/schema.js

```diff
@@ -0,0 +1,16 @@
+schema
```

Comment:
is this functional at all? it doesn't look like js, is it some weird YAML thing?

Reply:
removed